en/snap/with-plugins.md

```diff
@@ -420,313 +487,436 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
+| Exp Month   | 01                  |
+| Exp Year    | 2025                |
+| OTP/3DS     | 112233              |
 
+For more details, refer to [Testing Payments on Sandbox](/en/technical-reference/sandbox-test).
 2. Examine a few points below to ensure plugin is installed and performs properly.
 
-Check Point| Expected Result |When Unexpected Result Occurs
---- | --- | ---
-Check order status in CMS back end|  Order status recorded in backend| Check endpoint/payment notification URL setting in MAP.<br> Check if your CMS/notification url can be accessed from public internet properly
-Merchant receives email notification|  Receive notification|  Check MAP menu setting - email notification
-Customer receives email notification| Receive notification|  Check MAP menu setting - email notification
+| Check Point                 | Error                                     | Troubleshooting                                              |
+| --------------------------- | ----------------------------------------- | ------------------------------------------------------------ |
+| Order status in CMS backend | Order status not recorded in the backend. | Check endpoint/Payment Notification URL setting on MAP.<br> Check if your CMS/Notification URL is publicly accessible. |
+| Merchant email notification | Notification not received.                | Check Email Notifications settings on MAP. For more details, refer to [Configuring Email Notifications](/en/after-payment/dashboard-usage.md#configuring-email-notifications). |
+| Customer email notification | Notification not received.                | Check Email Notifications settings on MAP. For more details, refer to [Configuring Email Notifications](/en/after-payment/dashboard-usage.md#configuring-email-notifications). |
 
 #### Payment Example
+
 ![Opencart Payment Test](./../../asset/image/opencart-pay-show.gif)
 
 </article>
 </details>
+
 <hr><br><br>
 
 ### WHMCS
 <hr>
 
-#### Requirements:
-   * WHMCS v5.3.12 - v7.x or greater (Tested up to WHMCS v7.6 - running well)
-   * PHP version 5.4 or greater
-   * MySQL version 5.0 or greater
-   * Midtrans plugin for WHMCS [ [Github](https://github.com/veritrans/SNAP-whmcs) | [Zip](https://github.com/veritrans/SNAP-whmcs/archive/master.zip) ].
-
-#### Installation:
-   1. Download the modules from the link above.
-   2. Extract **Whmcs-master.zip** file you have previously downloaded.
-   3. Upload & merged module folder that you have extracted into your WHMCS directory. **Installation & Configuration**
-   4. Access your WHMCS admin page.
-   5. Go to `Setup` **->** `Payments` **->** `Payment Gateways` menu.
-   6. Click **Midtrans** payment method, then you will be redirected to configuration page.
-   7. Fill the input as instructed on the screen. Click **Save Changes**.
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+   * WHMCS v5.3.12 - v7.x or later (Tested up to WHMCS v7.6 - running well)
```

Comment:
> (Tested up to WHMCS v7.6 - running well)

This also inconsistencies that we have, that you can fix, the sentence for `Tested up to` is not consistent with other instruction

Reply:
I will incorporate these changes.